Planomatic/App.xaml.cs

```diff
@@ -38,16 +33,14 @@ public void UpdateProgress(bool state)
 
         private void Application_Startup(object sender, StartupEventArgs e)
         {
+            // If we are load with a .plano file - load config, otherwise load placeholder
             String[] arguments = Environment.GetCommandLineArgs();
-
             if (arguments.GetLength(0) > 1)
             {
                 if (arguments[1].EndsWith(".plano"))
                 {
                     string filePathFormMainArgs = arguments[1];
                     CurrentConfig.LoadConfig(filePathFormMainArgs);
-                    CurrentConfig.LastConfigFilename = filePathFormMainArgs;

```

Comment:
Note: this is removed as it's already handled (saved) inside

 public bool LoadConfig(string configFilePath)
...
 LastConfigFilename = configFilePath;
                _lastConfigSet = true;